src/components/ConfirmationModalDialog/ConfirmationModalDialog.stories.js

```diff
@@ -126,9 +126,114 @@ const withCustomContent = () => ({
   `,
 });
 
+const withDismissButtonDisabled = () => ({
+  components: {
+    ConfirmationModalDialog,
+  },
+  data() {
+    return {
+      isShowing: false,
+    };
+  },
+  computed: {
+    showHideTitle() {
+      if (this.isShowing) { return 'Hide'; }
+      return 'Show';
+    },
+  },
+  methods: {
+    onDismiss: action('Dismissed!'),
+    onConfirm: action('Confirmed!'),
+    onClose: action('Closed!'),
+    toggleIsShowing() {
+      this.isShowing = !this.isShowing;
+    },
+  },
+  template: `
+    <div style="margin: 10px 50px 10px 50px;">
+      <h2><strong>ConfirmationModalDialog:</strong>&nbsp;Custom Content with disabled Dismiss Button.</h2>
+      <hr>
+      <div style="margin-top: 20px; width: 100px">
+        <ConfirmationModalDialog v-model="isShowing"
+                                 title="Custom Content with disabled Dismiss Button Example"
+                                 confirm-button-text="Got it"
+                                 dismiss-button-text="Never mind"
+                                 @dismiss="onDismiss"
+                                 @confirm="onConfirm"
+                                 @close="onClose"
+                                 dismissButtonDisabled
```

Comment:
(nit) this should be: `dismiss-button-disabled` because in HTML markup, we must use `kebab-case` (this is mostly due to HTML attributes being case-insensitive). Component names and `@` event handlers are fine with `MixedCase`, only the prop attributes require `kebab-case`. No need to change this now (because I saw it was already merged) however please do keep it in mind for next time 😉 

Similar comment for line: 215, below.

Reply:
😢